src/convolution_api.cpp

```diff
@@ -218,7 +218,8 @@ miopenConvolutionCKBackwardWeightsGetWorkSpaceSize(const miopenAlphaBetaCase_t a
         case miopenInt8:
         case miopenFloat8:
         case miopenBFloat8: byte_size = 4; break;
-        case miopenDouble: byte_size = 8;
+        case miopenDouble:
+        case miopenInt64: byte_size = 8; break;
```

Comment:
@bghimireamd Please take a look at this function. Is everything fine with `miopenHalf`, `miopenBFloat16`, `miopenInt8`, `miopenFloat8` and `miopenBFloat8`. In my superficial opinion, `byte_size` should have different values for these cases.

Reply:
Yes, this is correct. This byte size is for the accumulator type. The accumulator types for [`miopenHalf`, `miopenBFloat16`, `miopenInt8`, `miopenFloat8` and `miopenBFloat8`] is of byt_size 4. But for `miopenDouble` and `miopenInt64` will be byte_size 8. So this code make sense.